homeassistant/components/freebox/config_flow.py

```diff
@@ -102,7 +106,64 @@ async def async_step_link(self, user_input=None):
 
         return self.async_show_form(step_id="link", errors=errors)
 
-    async def async_step_import(self, user_input=None):
+    async def async_step_permissions(self, user_input=None) -> FlowResult:
```

Comment:
I don't see when this step is called, I think it's not.

Here it requires the user to accept the home permission, even if the Freebox model does not have Home capabilities (not Delta model).
1. Should we check the router model before forcing the Home permission ?
2. Should we force adding the permission for adding Home in the config flow as we already prevent the user in `router.update_home_devices` and in the documentation ?
3. Should we propose the user to use Home devices earlier in the config flow and then check permissions only if he accepts ?

I am good with option 3 or option 1 + 3 (propose only if model has Home capabilities), with 2 because if we are doing that we can remove the check in `router.update_home_devices`.

Reply:
Also, you just added this check.
Why ?

Reply:
it was added to separate camera and home devices... but after tests... enable home devices overlay camera permission....  I'll delete it